cookbooks/bcpc/recipes/serial_console.rb

```diff
@@ -0,0 +1,71 @@
+# declare a placeholder template
+template "/etc/init/ttySx.conf" do
+    source "ttySx.erb"
```

Comment:
Can we keep indentation to 2 spaces to keep it in align with all other recipes we have?